File: csharp/TV.cs
// csharp/TV.cs
namespace ECommerce
{
    public class TV : ShippableProduct
    {
        public TV(string name, double price, int quantity, double weight)
            : base(name, price, quantity, weight)
        {
        }

        public override bool IsExpired()
        {
            return false;
        }
    }
}

File: javascript/TV.js
// javascript/TV.js
import { ShippableProduct } from './ShippableProduct.js';

export class TV extends ShippableProduct {
    constructor(name, price, quantity, weight) {
        super(name, price, quantity, weight);
    }

    isExpired() {
        return false;
    }
}
